### PR TITLE
feat: sale price too low toast error

### LINKED ISSUE #226: Handle the new sale price error from decentraland-transactions
Add a new case to the [meta transaction error switch](https://github.com/decentraland/decentraland-dapps/blob/f3f86f1cf72f222aa1330394c299669f039292c0/src/modules/toast/sagas.ts#L76) that handles the new sale price error

### PR DESCRIPTION
**Pull request by nicosantangelo**
Shows a toast modal error when the price of the meta transaction sale is too low
Closes #226

### CODE REVIEW

File: src/modules/toast/sagas.spec.ts [hunk 1]
```diff
@@ -77,6 +80,18 @@ describe('when handling a meta-transaction error', () => {
       ).silentRun()
     })
   })
+
+  describe('when the error code is SALE_PRICE_TOO_LOW', () => {
+    it('should show an unknown error toast', () => {
```
Comment (LautaroPetaccio) [inline src/modules/toast/sagas.spec.ts:85]:
```suggestion
    it('should show a sales price too low error toast', () => {
```

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]
It seems that the `package-lock.json` changed, but there are no dependencies being updates in the `package.json`, I think that you're missing them.

**Comment by nicosantangelo** [comment]
Thanks for the review peta <3

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**